lineQueueClient/src/components/Signup/Signup.js

```diff
@@ -1,8 +1,9 @@
 import React, {Component} from 'react'
-import {Text, View, TextInput, Button} from 'react-native';
+import {Text, View, TextInput, TouchableHighlight} from 'react-native';
 import Header from '../Header/Header'
+import globals from '../../styles/globals.style'
 import signupStyle from './Signup.style'
-import { Actions } from 'react-native-router-flux'
+import {Actions} from 'react-native-router-flux'
 
 export default class Signup extends Component {
```

Comment:
Add documentation
short explanation about component, similar to `Header` component